backend/app/controllers/uoas.js

```diff
@@ -68,26 +68,26 @@ module.exports = {
     },
 
     insert: function (req, res, next) {
-        //TODO: Fix test to pass for this
-        console.log('I GOT IN HERE');
         var thunkQuery = req.thunkQuery;
-        console.log('I EVEN GOT HERE and body is: ' + req.body.subjects)
-        var uoas = req.body.subjects.map((subject) => subject.name);
-        console.log('I DIDN"T EVEN GET HERE')
-        var sqlString = "'"+uoas.toString().replace(/'/g, "''").replace(/,/g, "','")+"'";
 
-        co(function* () {
-            console.log('I ENTERED SUCCESSFULLY');
-            console.log('UOA TYPE IS: ' + req.body.unitOfAnalysisType);
+        // Verify that the body contains
+        if (req.body.subjects) {
+            if (!Array.isArray(req.body.subjects)) {
+                req.body.subjects = [{name: req.body.subjects}];
+            }
```

Comment:
@cporter-amida This function expects an array of objects to be passed in otherwise it breaks. So we ensure that an array is passed in, if not, we convert what is passed in into an array. I think it could have been done differently however that will require a reformatting of the entire function beyond the scope of this ticket. Would need to talk to @jmfadeley about it. I'm ticketing it somewhere else. This was really just to get the tests to pass.